### ISSUE TITLE
Unnecessary chown during wizard run throws error while everything works fine

### ISSUE THREAD

**Issue by dgoetz** [Icinga/icinga2#8743]
## Describe the bug

For a training we have pre-installed Icinga 2 using Puppet which sets file permssions for zones.conf to:

```
-rw-r-----. 1 root icinga 220 Apr 28 14:50 /etc/icinga2/zones.conf
```

If the command `icinga2 node wizard` is then run it complains about a failed chown:

```
Default global zones: global-templates director-global
Do you want to specify additional global zones? [y/N]: 
critical/cli: chown() failed with error code 1, "Operation not permitted"
```

But the wizard does copy the file over and creates a new one without errors.

## To Reproduce

Just set the file permissions / ownership and run the wizard.

1. `chown root.icinga /etc/icinga2/zones.conf && chmod 0640 /etc/icinga2/zones.conf`
2. `icinga2 node wizard`

Can be repeated as often as necessary.

## Expected behavior

No chown is executed and / or no error is caused in this case because it is not required to move the file to .orig and create a new one. 

## Your Environment

* Version used (`icinga2 --version`): 2.12.3
* Operating System and version: CentOS 7

**Comment by Al2Klimov** [comment]
Indeed...

```
root@453b889050c7:/# mkdir -m 0777 /sandbox
root@453b889050c7:/# ls -la /sandbox
total 8
drwxrwxrwx 2 root root 4096 Apr 29 17:05 .
drwxr-xr-x 1 root root 4096 Apr 29 17:05 ..
root@453b889050c7:/# touch /sandbox/mine
root@453b889050c7:/# ls -l /sandbox
total 0
-rw-r--r-- 1 root root 0 Apr 29 17:06 mine
root@453b889050c7:/# chmod 0600 /sandbox/mine
root@453b889050c7:/# ls -l /sandbox
total 0
-rw------- 1 root root 0 Apr 29 17:06 mine
root@453b889050c7:/# su -ls /bin/bash nobody
su: warning: cannot change directory to /nonexistent: No such file or directory
nobody@453b889050c7:/$ mv /sandbox/mine /sandbox/yours
nobody@453b889050c7:/$ ls -la /sandbox
total 8
drwxrwxrwx 2 root root 4096 Apr 29 17:07 .
drwxr-xr-x 1 root root 4096 Apr 29 17:05 ..
-rw------- 1 root root    0 Apr 29 17:06 yours
nobody@453b889050c7:/$ chown nobody /sandbox/yours
chown: changing ownership of '/sandbox/yours': Operation not permitted
nobody@453b889050c7:/$
```

**Comment by Al2Klimov** [comment]
Indeed...

```
root@05181121ae10:/# mkdir -m 0777 /sandbox
root@05181121ae10:/# touch /sandbox/a
root@05181121ae10:/# chmod 0600 /sandbox/a
root@05181121ae10:/# su -ls /bin/bash nobody
su: warning: cannot change directory to /nonexistent: No such file or directory
nobody@05181121ae10:/$ chown nobody /sandbox/a
chown: changing ownership of '/sandbox/a': Operation not permitted
nobody@05181121ae10:/$ touch /sandbox/b
nobody@05181121ae10:/$ mv /sandbox/b /sandbox/a
mv: replace '/sandbox/a', overriding mode 0600 (rw-------)? y
nobody@05181121ae10:/$ ls -la /sandbox
total 8
drwxrwxrwx 2 root   root    4096 Apr 29 17:11 .
drwxr-xr-x 1 root   root    4096 Apr 29 17:10 ..
-rw-r--r-- 1 nobody nogroup    0 Apr 29 17:11 a
nobody@05181121ae10:/$
```

**Comment by chrwahl** [comment]
Following https://github.com/Icinga/icinga2/issues/7854#issuecomment-783473100 I did `chown -R nagios:nagios /etc/icinga2/features-available/` on Ubuntu 20.04
